docs/api/tutorial-uploading-photo-to-a-product.md

```diff
@@ -6,9 +6,15 @@ Be sure to also read the [introduction to the API](./index.md)
 
 ## Points to consider before uploading photos
 
+### Why photos are important on Open Food Facts
+* Photos are source and proof of data.
+* They build trust in your edit.
+* They are the fastest and laziest way for your users to contribute
```

Comment:
I don't know if "lazy" is the best way to word it. I think it usually has a pretty negative connotation? Maybe a native speaker could have a look?

Reply:
easiest might be more consensual ?